pkg/expect/expect.go

```diff
@@ -174,63 +213,90 @@ func (ep *ExpectProcess) LineCount() int {
 	return ep.count
 }
 
+// ExitCode returns the exit code of this process.
+// This will return the correct exit code after Close/Stop has been called, otherwise it returns math.MinInt
+func (ep *ExpectProcess) ExitCode() int {
+	ep.mu.Lock()
+	defer ep.mu.Unlock()
+
+	if ep.cmd == nil {
+		return ep.exitCode
+	}
+
+	return math.MinInt
```

Comment:
this is weird decision. would prefer to return an error or some known error code like 255

Reply:
agreed, will change this